browser/brave_rewards/android/brave_rewards_native_worker.cc

```diff
@@ -69,12 +69,39 @@ void BraveRewardsNativeWorker::Destroy(JNIEnv* env) {
   delete this;
 }
 
-void BraveRewardsNativeWorker::CreateRewardsWallet(JNIEnv* env) {
+std::string BraveRewardsNativeWorker::StringifyResult(
+    ledger::mojom::CreateRewardsWalletResult result) {
+  switch (result) {
+    case ledger::mojom::CreateRewardsWalletResult::kSuccess:
+      return "success";
+    case ledger::mojom::CreateRewardsWalletResult::kWalletGenerationDisabled:
+      return "wallet-generation-disabled";
+    case ledger::mojom::CreateRewardsWalletResult::kGeoCountryAlreadyDeclared:
+      return "country-already-declared";
+    case ledger::mojom::CreateRewardsWalletResult::kUnexpected:
+      return "unexpected-error";
+  }
+}
```

Comment:
Can we somehow avoid copying this from `brave_rewards_api.cc`?

Reply:
@szilardszaloki i thought of including it in rewards_service_impl but then we needed to update desktop implementation. so i didn't touch it

Reply:
@zenparsing should we keep as is for now ?